docs/2.10.0/docs/daml/intro/9_Dependencies.rst

```diff
@@ -56,9 +56,12 @@ We can see this in action. When a DAR file gets deployed to a ledger, not all me
 
 You'll notice two things. Firstly, a lot of the dependencies have lost their names, they are now only identifiable by hash. We could of course also create a second project ``intro7-1.0.0`` with completely different contents so even when name and version are available, package hash is the only safe identifier.
 
-That's why over the Ledger API, all types, like templates and records are identified by the triple ``(entity name, module name, package hash)``. Your client application should know the package hashes it wants to interact with. To aid that, ``inspect-dar`` also provides a machine-readable format for the information it emits: ``daml damlc inspect-dar --json assets_ledger.dar``. The ``main_package_id`` field in the resulting JSON payload is the package hash of our project.
+That's why over the Ledger API, all types, like templates and records are identified by the triple ``(entity name, module name, package hash)`` which refers to the version of the package that was used to create them. Your client application should know the package hashes it wants to interact with. To aid that, ``inspect-dar`` also provides a machine-readable format for the information it emits: ``daml damlc inspect-dar --json assets_ledger.dar``. The ``main_package_id`` field in the resulting JSON payload is the package hash of our project.
 
-Secondly, you'll notice that all the ``*.daml``, ``*.hi`` and ``*.hie`` files are gone. This leads us to data dependencies.
+Starting with 2.10, additional a so called package name is available, which is a human readable name for the package. Your client application can now also be based off the package names it wants to interact with. The participant guarantees that all packages referring
```

Comment:
Package name has been available for a while. its just only used in 2.10, and only in LF1.17 (i.e. not by default)
"Starting with 2.10, clients can interact with packages via their package name, a human readable name defined in the `daml.yaml`. The participant .... package is upgraded. This functionality is only available when Smart Contract Upgrading is enabled, which is covered here"
Perhaps something like this?